cmd/osmosisd/cmd/root.go

```diff
@@ -924,8 +923,6 @@ func txCommand(moduleBasics module.BasicManager) *cobra.Command {
 		authcmd.GetDecodeCommand(),
 	)
 
-	// UNFORKING v2 TODO: Auto CLI claims we can remove this, but if we do, then the legacy proposal sub commands will not be available.
```

Comment:
Did you confirm the legacy proposal sub commands still show?

Reply:
```
λ ghost osmosis → λ git chore/unforking-todos* → make install
Go version: 1.22
GOWORK=off go install -mod=readonly -tags "netgo ledger" -ldflags '-X github.com/cosmos/cosmos-sdk/version.Name=osmosis -X github.com/cosmos/cosmos-sdk/version.AppName=osmosisd -X github.com/cosmos/cosmos-sdk/version.Version=adam/sqs-v26-testing-68-g985a6ede8 -X github.com/cosmos/cosmos-sdk/version.Commit=985a6ede848f2cc1048ba92ddf180eee222ed637 -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" -w -s' -trimpath github.com/osmosis-labs/osmosis/v25/cmd/osmosisd

λ ghost osmosis → λ git chore/unforking-todos* → osmosisd tx gov submit-legacy-proposal --help
Submit a legacy proposal along with an initial deposit.
Proposal title, description, type and deposit can be given directly or through a proposal JSON file.

Example:
$ osmosisd tx gov submit-legacy-proposal --proposal="path/to/proposal.json" --from mykey
```

Was there another command that you noticed missing?

Reply:
ah soz, yeah they're missing

Reply:
Can you find these commands in the v25.x release line? :confused:

Reply:
> Did you confirm the legacy proposal sub commands still show?

I'm struggling here a little, can you add one that I can `copy/pasta`?

Reply:
This also makes e2e fail :cry:

Reply:
Do `osmosisd q gov submit-legacy-proposal -h` and you will see these Available Commands either missing or present:

```
Available Commands:
  cancel-software-upgrade                    Cancel the current software upgrade proposal
  create-cl-pool-and-cfmm-link               Submit a create clpool and link to cfmm proposal
  create-concentratedliquidity-pool-proposal Submit a create concentrated liquidity pool proposal
  create-groups-proposal                     Submit a create groups proposal
  denom-pair-taker-fee-proposal              Submit a denom pair taker fee proposal
  ibc-upgrade                                Submit an IBC upgrade proposal
  migrate-cw-pool-contracts                  Submit a migrate cw pool contracts proposal
  param-change                               Submit a parameter change proposal
  remove-superfluid-assets-proposal          Submit a superfluid asset remove proposal
  replace-migration-records-proposal         Submit a replace migration record proposal
  replace-pool-incentives                    Submit a full replacement to the records for pool incentives
  set-scaling-factor-controller-proposal     Submit a set scaling factor controller proposal
  set-superfluid-assets-proposal             Submit a superfluid asset set proposal
  software-upgrade                           Submit a software upgrade proposal
  tick-spacing-decrease-proposal             Submit a tick spacing decrease proposal
  update-client                              Submit an update IBC client proposal
  update-fee-token                           Submit a update fee token record proposal
  update-migration-records-proposal          Submit a update migration record proposal
  update-pool-incentives                     Submit an update to the records for pool incentives
  update-unpool-whitelist                    Update unpool whitelist proposal
  upload-code-id-and-whitelist               Submit an upload code id and whitelist proposal
  ```

Reply:
It fails because e2e uses these legacy commands, I have an issue to move away from legacy but alas no time

Reply:
ah ye, they are missing, I've added them back now, the issue we have now is now some commands are doubled, but only the commands with:

```
λ ghost osmosis → λ git chore/unforking-todos → osmosisd tx | grep - 
  ibc-transfer           IBC fungible token transfer transaction subcommands
  ibc-transfer           IBC fungible token transfer transaction subcommands
  ibc-wasm               IBC wasm manager module transaction subcommands
  ibc-wasm               IBC wasm manager module transaction subcommands
  interchain-accounts    IBC interchain accounts transaction subcommands
  interchain-accounts    IBC interchain accounts transaction subcommands
```

Reply:
https://github.com/osmosis-labs/osmosis/issues/8599

I created this issue to track, should we merge this and deal with that issue in a vacuum